test/cli/test_run.py

```diff
@@ -151,6 +151,63 @@ def test_main_with_logging_config(command, tmpdir):
     assert "test.sample - INFO - ping!" in capture_file.read()
 
 
+def test_multiple_service_modules(command):
+
+    with patch("nameko.cli.run.main") as run:
+
+        command(
+            "nameko",
+            "run",
+            "test.foo",
+            "test.bar"
+        )
+
+        assert run.call_args_list == [call(("test.foo", "test.bar"), None)]
+
+
+def test_parse_config_before_service_import(command, tmpdir, add_to_sys_path):
+
+    config_file = tmpdir.join("config.yaml")
+    config_file.write(dedent(
+        """
+        NAME: dynamic
+        """
+    ))
+
+    service_file = tmpdir.join("service.py")
+    service_file.write(dedent(
+        """
+        from nameko import config
+        from nameko.testing.services import dummy
+
+        class Service:
+            name = config.get("NAME", "notfound")
```

Comment:
nice, this test is actually a use case somebody asked for in the discussion board, or in one of issues here on github .)